### PR TITLE
🎨 Format `logType` with camel case

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/server/api/routes/users/v2/nfts/display-state.js [hunk 1]
```diff
@@ -59,17 +59,17 @@ router.post(
...
           payload.featuredNFTClassIds = FieldValue.arrayUnion(classId);
           payload.hiddenNFTClassIds = FieldValue.arrayRemove(classId);
-          logType = 'dashboard_nft_display_state_feature';
+          logType = 'dashboardNFTDisplayStateFeature';
           break;
         case NFT_DISPLAY_STATE.HIDDEN:
           payload.featuredNFTClassIds = FieldValue.arrayRemove(classId);
           payload.hiddenNFTClassIds = FieldValue.arrayUnion(classId);
-          logType = 'dashboard_nft_display_state_hide';
+          logType = 'dashboardNFTDisplayStateHide';
```
Comment (williamchong) [inline src/server/api/routes/users/v2/nfts/display-state.js:67]:
```suggestion
          logType = 'DashboardNFTDisplayStateHide';
```

File: src/server/api/routes/users/v2/nfts/display-state.js [hunk 2]
```diff
@@ -59,17 +59,17 @@ router.post(
...
           break;
         case NFT_DISPLAY_STATE.DEFAULT:
           payload.featuredNFTClassIds = FieldValue.arrayRemove(classId);
           payload.hiddenNFTClassIds = FieldValue.arrayRemove(classId);
-          logType = 'dashboard_nft_display_state_reset';
+          logType = 'dashboardNFTDisplayStateReset';
```
Comment (williamchong) [inline src/server/api/routes/users/v2/nfts/display-state.js:72]:
```suggestion
          logType = 'DashboardNFTDisplayStateReset';
```

**Review by williamchong — CHANGES_REQUESTED** [review]

**Comment by nwingt** [comment]
@WeiJun0827 @williamchong Why changing to camel case?

**Comment by williamchong** [comment]
> @WeiJun0827 @williamchong Why changing to camel case?

 underscore for ga, Camel for es

**Pull request merged**